fission/src/mirabuf/MirabufSceneObject.ts

```diff
@@ -695,6 +764,13 @@ class MirabufSceneObject extends SceneObject implements ContextSupplier {
 
         return data
     }
+
+    private RobotCollision(collision: Jolt.BodyID) {
```

Comment:
```suggestion
    private RecordRobotCollision(collision: Jolt.BodyID) {
```

I think it's mostly a standard that functions with side effects (non getters) should have verbs just to make it clear what action they're doing